rasa_core/broker.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+
+import pika
+
+logger = logging.getLogger(__name__)
+
+
+class EventBroker(object):
```

Comment:
This should rather be called `EventChannel` (it is not a broker from a technical point of view) - I think the correct way to call it would be an event channel connection but that is a horrible name. the broker is kafka.

Accordingly, I'd call `process` smth like `emit` or `publish`